unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/HUDController.cs

```diff
@@ -148,7 +150,8 @@ public enum HUDElementID
         EMAIL_PROMPT = 21,
         USERS_AROUND_LIST_HUD = 22,
         GRAPHIC_CARD_WARNING = 23,
-        COUNT = 24
+        BUILD_MODE = 24,
```

Comment:
We are gonna have conflict with this. Not sure if you are using this ID in kernel, if so dont foret to change it!

Reply:
No, I don't use it in kernel. I just followed its implementation and saw that the only one using it was the  HUDTests, and the builder ( `DCLBuilderBridge` ) to set the visibility to false of all HUD elements. So in order to keep it working, I just added mine.

I don't know if this is causing any problem on the kernel side, if so, please let me know

Reply:
Sounds good!